src/Algebra/Graph.hs

```diff
@@ -1189,6 +1193,10 @@ matchR e v p = \x -> if p x then v x else e
 -- their buildR form.
 "bindR/bindR" forall c f g.
     composeR (composeR c f) g = composeR c (f.g)
+
+-- Remove identity (which can appear in the rewriting of bindR)
```

Comment:
This rule doesn't actually "remove" identity -- it rewrites it into a simpler/more efficient form.

Reply:
Fixed ^^